src/TodoForm.jsx

```diff
@@ -1,9 +1,23 @@
-function TodoForm() {
+import { useRef } from 'react';
+
+function TodoForm({ onAddTodo }) {
+  const todoTitleInput = useRef("");
+
+  function handleAddTodo(event) {
+    event.preventDefault();
+    const title = event.target.title.value;
+    event.target.title.value = "";
```

Comment:
This works just fine but it's better practice to reset your inputs after all logic is complete. Once your codebase grows this smalls difference can create bugs.